src/siema.js

```diff
@@ -533,9 +533,15 @@ export default class Siema {
       // if dragged element is a link
       // mark preventClick prop as a true
       // to detemine about browser redirection later on
-      if (e.target.nodeName === 'A') {
-        this.drag.preventClick = true;
-      }
+
+      let target = e.target
+      do {
+        if (target.nodeName === 'A') {
```

Comment:
How can i edit siema.js from src, and be able to use it?

If i use the unminified siema it doesnt work

I want to compile a new siema with your modification, im using v1.51 but have the link-open-bug when sliding link content